### DISCUSSION TITLE
Docklike add a blank space after start menu

### DISCUSSION THREAD

**Discussion by ak1dd** [ramensoftware/windhawk-mods#1168] [General]
I use Nilesoft Shell to have custom system-wide context menus.  The most common way to invoke its settings is to right-click on the taskbar on an empty space and it appears as an option on the menu.     I use  DockLike and there is no way to right-click anywhere that is a blank space.   Is there a way I could have a blank space between the start menu and the first app icon on the taskbar?  I think this would be visually appealing and provide that ability.

**Comment by m417z** [comment]
It was discussed a bit here: https://github.com/ramensoftware/windows-11-taskbar-styling-guide/issues/101#issuecomment-2366726836.
I didn't try it, but you can experiment with UWPSpy and see if that's possible.
